po/zh_CN.po

```diff
@@ -814,6 +814,83 @@ msgid ""
 "*** Deprecated *** Whether to remove stopped resources from the executor"
 msgstr "***不推荐***是否从pacemaker-execd 守护进程中清除已停止的资源"
 
+#: tools/crm_resource.c:257
+#, c-format
+msgid "Aborting because no messages received in %d seconds"
+msgstr "中止，因为在%d秒内没有接收到消息"
+
+#: tools/crm_resource.c:908
+#, c-format
+msgid "Invalid check level setting: %s"
+msgstr "无效的检查级别设置：%s"
+
+#: tools/crm_resource.c:992
+#, c-format
+msgid ""
+"Resource '%s' not moved: active in %d locations (promoted in %d).\n"
+"To prevent '%s' from running on a specific location, specify a node.To "
+"prevent '%s' from being promoted at a specific location, specify a node and "
+"the --promoted option."
+msgstr ""
+"资源'%s'未移动：在%d个位置运行（其中在%d个位置为主实例）\n"
+"若要阻止'%s'在特定位置运行，请指定一个节点。若要防止'%s'在指定位置升级，指定"
+"一个节点并使用--promoted选项"
+
+#: tools/crm_resource.c:1003
+#, c-format
+msgid ""
+"Resource '%s' not moved: active in %d locations.\n"
+"To prevent '%s' from running on a specific location, specify a node."
+msgstr ""
+"资源%s未移动，在%d个位置运行\n"
+"若要防止'%s'运行在特定位置，指定一个节点"
+
+#: tools/crm_resource.c:1078
+#, c-format
+msgid "Could not get modified CIB: %s\n"
+msgstr "无法获得修改的CIB：%s\n"
+
+#: tools/crm_resource.c:1112
+msgid "You need to specify a resource type with -t"
+msgstr "需要使用-t指定资源类型"
+
+#: tools/crm_resource.c:1155
+#, c-format
+msgid "No agents found for standard '%s'"
+msgstr "没有发现指定的'%s'标准代理"
+
+#: tools/crm_resource.c:1158
+#, fuzzy, c-format
+msgid "No agents found for standard '%s' and provider '%s'"
+msgstr "没有发现指定的'%s'标准代理"
```

Comment:
Somehow this got imprecise anymore and basically it doesn't match the original format.